store/sqlstore/upgrade.go

```diff
@@ -688,6 +688,9 @@ func UpgradeDatabaseToVersion512(sqlStore SqlStore) {
 		sqlStore.CreateColumnIfNotExistsNoDefault("Schemes", "DefaultChannelGuestRole", "text", "VARCHAR(64)")
 		sqlStore.GetMaster().Exec("UPDATE Schemes SET DefaultTeamGuestRole = '', DefaultChannelGuestRole = ''")
 
+		// Saturday, January 24, 2065 5:20:00 AM GMT. To remove all personal access token sessions.
+		sqlStore.GetMaster().Exec("DELETE FROM Sessions WHERE ExpiresAt > 3000000000000")
```

Comment:
The only way to identify the sessions that exist because of access tokens is their excessive expiry time. Not completly happy with this. Open to suggestions.

Reply:
Is the purpose here to just have all the sessions re-create so that they're correctly labelled as bot sessions?

Reply:
@jwilander Yes. Otherwise if you have the `EnableUserAccessTokens` set to false, your existing sessions won't work.

Reply:
Ok I don't think there's any other way around this